src/algebra/big_operators/multiset.lean

```diff
@@ -322,6 +338,14 @@ lemma prod_eq_one_iff [canonically_ordered_monoid α] {m : multiset α} :
   m.prod = 1 ↔ ∀ x ∈ m, x = (1 : α) :=
 quotient.induction_on m $ λ l, by simpa using list.prod_eq_one_iff l
 
+/-- Slightly more general version of `prod_eq_one_iff` for a non-ordered `monoid` -/
+@[to_additive "Slightly more general version of `sum_eq_zero_iff` for a non-ordered `monoid`"]
```

Comment:
So that this links in the documentation:
```suggestion
/-- Slightly more general version of `multiset.prod_eq_one_iff` for a non-ordered `monoid` -/
@[to_additive "Slightly more general version of `multiset.sum_eq_zero_iff` for a non-ordered `add_monoid`"]
```